demo/docker-compose.yaml

```diff
@@ -50,17 +52,41 @@ services:
       [ "--node-id", "3"
       , "--api-port", "4003"
       , "--port", "5003"
+      , "--monitoring-port", "6003"
       , "--peer", "localhost@5001"
       , "--peer", "localhost@5002"
       , "--me", "/keys/carol.sk"
       , "--party", "/keys/alice.vk"
       , "--party", "/keys/bob.vk"
       ]
     network_mode: host # Required as mock-chain not configurable
-      
+
   mock-chain:
     image: inputoutput/hydra:mock-chain-latest
     build:
       context: ../
       target: mock-chain
     network_mode: host # Required as not configurable on node-side
+
+  prometheus:
+    image: prom/prometheus:latest
+    container_name: prometheus
+    ports:
+    - 9090:9090
+    command:
+    - --config.file=/etc/prometheus/prometheus.yml
+    volumes:
+    - ./prometheus.yml:/etc/prometheus/prometheus.yml:ro
+    depends_on:
+    - cadvisor
+
+  cadvisor:
+    image: gcr.io/cadvisor/cadvisor:latest
+    container_name: cadvisor
+    ports:
+    - 8080:8080
+    volumes:
+    - /:/rootfs:ro
```

Comment:
Not really confident about mounting `/` to a container :no_mouth: ?

Reply:
It's _read-only_ :) But yes, you're right, just copy/pasted example from prometheus, will fix it later if you don't mind.